AffirmSDK/AffirmCardInfoViewController.m

```diff
@@ -258,28 +250,7 @@ - (void)cancel:(id)sender
 
 - (void)cancelCard
```

Comment:
I don't know what's supposed to happen here but shouldn't a card be cancelled?

Reply:
It only dismisses a presented view controller, no matter with card data.